recurring_contract/models/contract_group.py

```diff
@@ -229,6 +229,7 @@ def generate_invoices(self):
         Context value async_mode set to False can force to perform
         the task immediately.
         """
+        self.ensure_one()
```

Comment:
This should not be here, the function handles recordset because of the `for group in self` loop.
```suggestion
```

Reply:
yes, i forgot to remove it. thanks